target_chains/ethereum/sdk/solidity/PythAggregatorV3.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: Apache 2
```

Comment:
MIT right?

Reply:
MIT lets us relicense under other licenses. I think it's better to use our standard license for uniformity.